service.go

```diff
@@ -555,22 +542,20 @@ func (client *Client) ListServicesWithProduct(product string, variables *Payload
 	return &q.Account.Services, nil
 }
 
-func NewTagArgs(tag string) TagArgs {
+func NewTagArgs(tag string) (TagArgs, error) {
```

Comment:
This would cause [code in the Terraform provider](https://github.com/OpsLevel/terraform-provider-opslevel/blob/main/opslevel/datasource_opslevel_services.go#L52) to break. Also in 4 other parts of the same repo

Reply:
Will put this in a separate PR